javascript/Dockerfile

```diff
@@ -0,0 +1,41 @@
+# syntax=docker.io/docker/dockerfile:1.4
+
+# build stage: includes resources necessary for installing dependencies
+
+# Here the image's platform does not necessarily have to be riscv64.
+# If any needed dependencies rely on native binaries, you must use 
+# a riscv64 image such as cartesi/node:19-jammy for the build stage,
+# to ensure that the appropriate binaries will be generated.
+FROM node:19-bullseye  as build-stage
+
+WORKDIR /opt/cartesi/dapp
+COPY . .
+RUN yarn
+RUN yarn build
+
+# runtime stage: produces final image that will be executed
+
+# Here the image's platform MUST be linux/riscv64.
+# Give preference to small base images, which lead to better start-up
+# performance when loading the Cartesi Machine.
+FROM --platform=linux/riscv64 cartesi/node:19-jammy-slim 
+
+LABEL io.cartesi.rollups.sdk_version=0.14.0
+LABEL io.cartesi.rollups.ram_size=128Mi
+
+RUN <<EOF
+apt-get update
+apt-get install -y --no-install-recommends busybox-static=1:1.30.1-7ubuntu3
+rm -rf /var/lib/apt/lists/*
+EOF
+
+COPY --from=sunodo/machine-emulator-tools:0.10.0-ubuntu22.04 / /
+ENV PATH="/opt/cartesi/bin:${PATH}"
+
+WORKDIR /opt/cartesi/dapp
```

Comment:
We could use non-root user for our final docker image .

Reply:
The user does not make a different, because in the end what we want is to export the docker image as a filesystem and run a cartesi machine from that